### PR TITLE
Accept team invite once signup

### LINKED ISSUE #1434: An new account accepting a team invite should be accepted by default
### Issue Summary

When a new user is created via team invite, after doing the onboarding, you need to go to team settings and accept the invite again. The expected behavior is to be already on the team on account creation.

### Steps to Reproduce

1. As user "A" invite a new team member
2. New user "B" should accept the invite and create an account
3. Notice how the invite is still "pending"

### PR DESCRIPTION
**Pull request by miguelnietoa**
## What does this PR do?

Fixes #1434

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [x] As user "A" invite a new team member
- [x] New user "B" should accept the invite and create an account
- [x] Now, the team invite is accepted.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code and corrected any misspellings
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: pages/api/auth/signup.ts [hunk 1]
```diff
@@ -69,5 +69,17 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     },
   });
 
+  // If user has been invitedTo a team, we accept the membership
```
Comment (juanhurtado10) [inline pages/api/auth/signup.ts:72]: @miguelnietoa may tests be added?

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]

File: pages/api/auth/signup.ts (hunk 1 above)
Comment (miguelnietoa) [reply pages/api/auth/signup.ts:72]: Of course, I can add some expectations in the Successful signup test. Working on that now!

**Review by juanhurtado10 — APPROVED** [review]
@miguelnietoa Awesome 👏 👏

**Comment by miguelnietoa** [comment]
This PR can be closed because the changes have already been merged into the main repo.
https://github.com/calcom/cal.com/pull/1634

**Pull request closed without merging**